eng/CodeCoverage.proj

```diff
@@ -20,8 +20,8 @@
       <_ReportGeneratorPath>$(PkgReportGenerator)\tools\net47\ReportGenerator.exe</_ReportGeneratorPath>
     </PropertyGroup>
 
-    <Message Importance="high" Text="&quot;$(_ReportGeneratorPath)&quot; -reports:$(ArtifactsDir)coverage\*.coverage -targetdir:$(ArtifactsDir)coverage -reporttypes:Cobertura" />
-    <Exec Command="&quot;$(_ReportGeneratorPath)&quot; -reports:$(ArtifactsDir)coverage\*.coverage -targetdir:$(ArtifactsDir)coverage -reporttypes:Cobertura" />
+    <Message Importance="high" Text="&quot;$(_ReportGeneratorPath)&quot; -reports:$(ArtifactsDir)\TestResults\**\*.cobertura.xml -targetdir:$(ArtifactsDir)coverage -reporttypes:Cobertura -sourcedirs:$(ArtifactsDir)/../" />
```

Comment:
Did you mean to leave this message behind?

If it's for debugging purposes only and want to keep it, I'd say importance can be lowered and this will still be available in the binlog

Reply:
I didn't write the initial one so I don't know if that was only for debugging purpose (cc @sharwell). But that's a good remark, thanks!